validator/client/metrics.go

```diff
@@ -258,23 +269,29 @@ func (v *validator) LogValidatorGainsAndLosses(ctx context.Context, slot types.S
 			startBalance := float64(v.startBalances[pubKeyBytes]) / gweiPerEth
 			percentNet := (newBalance - prevBalance) / prevBalance
 			percentSinceStart := (newBalance - startBalance) / startBalance
-			log.WithFields(logrus.Fields{
+
+			previousEpochSummaryFields := logrus.Fields{
```

Comment:
I suggest adding here Altair's Inactivity scores

Reply:
Thanks! I totally meant to do that!